content/lxd/getting-started-cli.md

```diff
@@ -147,10 +151,15 @@ Run the following command to specify the configuration options:
 
     sudo lxd init
 
-If you want to use the default options, add the `--auto` flag:
+**Quick test setup**
+
+To create a non-optimized test setup with default options, you can skip the configuration steps by adding the `--auto` flag:
 
     sudo lxd init --auto
 
+**Note:** The result will be slower and provides less functionality; especially the `dir storage backend` (which is used by default) will be slower and doesn't provide fast snapshots, fast copy/launch, quotas and optimized backups.
+
+If you want to use an optimized setup, we recommend to go through the configuration process instead.
```

Comment:
:thinking: Let me explain why I picked my version above; I think that the words "automatic" and "manual" are a bit confusing in this context, so I tried to avoid using them.

The reason is this:
- `manual` might be misunderstood as a way of configuration without `lxd init`
- `automatic` might be misunderstood as a way of automatically recognizing things, while in "reality" it is only a standard config that is applied.

As a longterm solution I would maybe suggest to rename the `--auto` function to something else, like `--default` (@stgraber).

---

So maybe I would change it to this:

`**Note:** The test setup will usually be slower and provide less functionality. Especially the dir storage backend (which is used by default) is slower and doesn't provide fast snapshots, fast copy/launch, quotas and optimized backups.`

Reply:
Maybe you should use the word "assisted" rather than "manual"?

Reply:
The reason for my suggestion was that you cannot use "slower" or "less" without stating what you are comparing to. Slower than what? Less functionality than what?

No strong opinion from my side on which terminology is best to distinguish "without --auto" from "with --auto". I like "interactive". Maybe just "Compared to the interactive configuration, the setup using the `--auto` flag will usually be slower..."?

Reply:
Yeah, interactive is how I've personally always referred to it so that works :)
We've also added the `--minimal` option now so in a few months once that's in both LXD 4.x and LXD 4.0.x, we can start making use of that here.